src/reducers/headerSections.js

```diff
@@ -61,6 +61,14 @@ export default handleActions(
       });
       return newState;
     },
+    [getLatestBlockFulfilled]: (state, {payload}) => {
+      const newState = deepCopyState(state);
```

Comment:
Right now `state` is an object that contains a field with a reference to the `sectionList` array. the array contains information for each of the header sections.
I have to use `deepCopyState` here because I am creating a state change on a deeply nested object, and I can't change state directly because react reducers are pure functions. Am I right to say that if we refactor the headerSection into separate reducers ala #149 (@debragail) we can do away with the `sectionList` array and just have shallow objects? So state for the lastBlockNumberHeader will just be like
{
lastBlockNumber: ""
}
or something like that..?

Reply:
yes i like this better.  I'd like to get rid of that sectionList.